portal.go

```diff
@@ -223,6 +223,16 @@ func (br *IMBridge) NewPortal(dbPortal *database.Portal) *Portal {
 	return portal
 }
 
+// NewPortalForBackfill creates a portal for the client-side bridge to backfill
+// into.
+func (br *IMBridge) NewPortalForBackfill(guid, threadID string) (*Portal, error) {
+	portal := br.loadDBPortal(nil, nil, guid)
+	portal.ThreadID = threadID
+	portal.Update(nil)
+	err := portal.CreateMatrixRoom(nil, nil)
```

Comment:
Passing the chat info from the client may be necessary since the cloud bridge probably won't know about groups. Also won't need to set the thread ID manually then since it'll be automatically updated from the chat info.